docs/rofl/app.mdx

```diff
@@ -36,220 +41,222 @@ chains).
-
-   ```shell
-   # Make sure to use the latest Sapphire Localnet docker image
-   docker pull ghcr.io/oasisprotocol/sapphire-localnet:latest
-   # Assuming you are running this command from the `rofl-oracle` directory
-   docker run -it -p8544-8548:8544-8548 -v .:/rofls ghcr.io/oasisprotocol/sapphire-localnet:latest
-   ```
-
-In a few moments, the Sapphire Localnet will spin up and automatically launch
-your ROFL inside the compute node. See [localnet][localnet] for more
-information.
-
-[localnet]: https://github.com/oasisprotocol/docs/blob/main/docs/build/tools/localnet.mdx
-
-[rofl-dev]: https://github.com/oasisprotocol/oasis-sdk/pkgs/container/rofl-dev
-
-```
-sapphire-localnet 2024-09-19-git2332dba (oasis-core: 24.2, sapphire-paratime: 0.8.2, oasis-web3-gateway: 5.1.0)
-
- * Detected ROFL bundle: /rofls/rofl-oracle.orc
- * Starting oasis-net-runner with sapphire...
- * Waiting for Postgres to start....
- * Waiting for Oasis node to start.....
- * Starting oasis-web3-gateway...
- * Bootstrapping network (this might take a minute)...
- * Waiting for key manager......
- * Populating accounts...
-
-Available Accounts
-==================
-(0) 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 (10000 TEST)
-(1) 0x70997970C51812dc3A010C7d01b50e0d17dc79C8 (10000 TEST)
-(2) 0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC (10000 TEST)
-(3) 0x90F79bf6EB2c4f870365E785982E1f101E93b906 (10000 TEST)
-(4) 0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65 (10000 TEST)
-
-Private Keys
-==================
-(0) 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
-(1) 0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
-(2) 0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a
-(3) 0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6
-(4) 0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a
-
-HD Wallet
-==================
-Mnemonic:       test test test test test test test test test test test junk
-Base HD Path:   m/44'/60'/0'/0/%d
-
- * Configuring ROFL /rofls/rofl-oracle.orc:
-   Enclave ID: 0+tTmlVjUvP0eIHXH7Dld3svPppCUdKDwYxnzplndLea/8+uR7hI7CyvHEm0soNTHhzEJfk1grNoBuUqQ9eNGg==
-   ROFL admin test:bob funded 10001 TEST
-   Compute node oasis1qp6tl30ljsrrqnw2awxxu2mtxk0qxyy2nymtsy90 funded 1000 TEST
-   App ID: rofl1qqn9xndja7e2pnxhttktmecvwzz0yqwxsquqyxdf
-
-WARNING: The chain is running in ephemeral mode. State will be lost after restart!
-
- * Listening on http://localhost:8545 and ws://localhost:8546. Chain ID: 0x5afd
- * Container start-up took 65 seconds, node log level is set to warn.
+## Define Containerized Services
+
+What the core of the ROFL app does is defined with services running in Podman
+containers defined in [the compose file]. Anything that can run as a bunch of
+containers can also be run as a ROFL app.
+
+For an easier start, we will use the `oasisprotocol/demo-rofl` container which
+is defined in [the `demo-rofl` repository] and implements [a trivial oracle] in
```

Comment:
Let's fix the oracle script to also `sleep` in case the price cannot be fetched.